bindings/node/src/index.ts

```diff
@@ -11,18 +11,22 @@ interface MongoCryptKMSRequest {
   readonly message: Buffer;
 }
 
-interface MongoCryptStatus {
+export interface MongoCryptStatus {
   type: number;
   code: number;
   message?: string;
 }
 
-interface MongoCryptContext {
+interface MongoCryptContextCtor {
+  new (...args: unknown[]): MongoCryptContext;
+}
```

Comment:
Can we mark this only for testing as well? Are there any types better than `unknown[]` worth adding? I would colocate it with the export but that's just an 🧅

Reply:
I moved it below and removed the types.  The constructor type might be necessary for testing but if it is, I'll add it then.  This isn't something we ever construct from js-land outside of tests.

One other option is to define it in the tests and just cast the constructor as the appropriate type?